src/apps/backend/modules/account/types.py

```diff
@@ -49,8 +49,9 @@ class Account:
     id: str
     first_name: str
     last_name: str
+    password: str
```

Comment:
This does not sound right.

We never want to populate password on `Account`. For that matter, we should never have raw password of user.